core-bundle/contao/library/Contao/TemplateTrait.php

```diff
@@ -132,8 +132,21 @@ public function addSchemaOrg(array $jsonLd): void
 
 	/**
 	 * @param iterable<string, string|int|bool|\Stringable|null>|string|self|null $attributes
+	 *
+	 * @deprecated Deprecated since Contao 5.3, to be removed in Contao 6;
+	 *             use attrs() instead.
 	 */
 	public function attr(HtmlAttributes|iterable|string|null $attributes = null): HtmlAttributes
+	{
+		trigger_deprecation('contao/core-bundle', '5.3', 'Using "%s()" has been deprecated and will no longer work in Contao 6. Use "attrs()" instead.', __METHOD__);
```

Comment:
```suggestion
		trigger_deprecation('contao/core-bundle', '5.3', 'Using "$this->attr()" in templates is deprecated and will no longer work in Contao 6. Use "$this->attrs()" instead.');
```

Otherwise the message will say `Using "Contao\TemplateTrait::attr()" is deprecated …` and no one uses the method this way.